cli/src/meta/types/interpreter_caller/v1.rs

```diff
@@ -55,18 +56,31 @@ impl TryFrom<Vec<u8>> for InterpreterCallerMeta {
     }
 }
 
+impl TryFrom<&[u8]> for InterpreterCallerMeta {
+    type Error = anyhow::Error;
```

Comment:
don't use anyhow for lib logic i think

Reply:
using anyhow for lib implementation was already implemented for structs/types of this repo before this PR, I just tried to keep it the same for newly added types/structs implementations